src/core_atmosphere/physics/physics_monan/module_bl_mixingscalars.F

```diff
@@ -0,0 +1,305 @@
+module module_pbl_scalars
+   use mpas_kind_types,only: kind_phys => RKIND
+   use mpas_pool_routines
+   !use mpas_atmphys_vars
+   implicit none
+
+   private
+   public :: driver_pbl_scalars
+   contains
+   !
+   !================================================================================================================= 
+   ! A routine to calculate the dry mixing in PBL for scalars not included in the MYNN routine.
+   ! This is strongly based on "mynn_mix_chem" routine in the file 'bl_mynn_subroutines.F'
+   ! Saulo R. Freitas - 19 Sep 2024 (send comments to saulo.freitas@inpe.br)
+   !================================================================================================================= 
+   subroutine driver_pbl_scalars (itimestep,diag_physics,configs,state,mesh,time_lev,its,ite,kts,kte,dt_pbl)
+   !=================================================================================================================     
+      !--- input arguments
+      type(mpas_pool_type),intent(in) :: configs
+      type(mpas_pool_type),intent(in) :: diag_physics
+      type(mpas_pool_type),intent(in) :: state
+      type(mpas_pool_type),intent(in) :: mesh
+      integer,intent(in) :: itimestep,time_lev
+      integer,intent(in) :: its,ite,kts,kte
+      real, intent(in) :: dt_pbl
+      
+      !--- local vars
+      integer, parameter :: max_nscalars = 100
+      integer :: i,j,k,initflag,nscalars,isc
```

Comment:
"DTN 4.7 - Variable names made up of more than one word must use the snake_case style, that is, the words in lowercase and the rest separated by underscores."

initflag = init_flag ?

Reply:
not solved